python_examples/simulation_flow.py

```diff
@@ -10,26 +10,26 @@
 
 # Method to set the seed at the start of the simulation, for reproducibility
 
-pe.routine.Simulation.set_random_seed(seed=42)
+# pe.routine.Simulation.set_random_seed(seed=42)
```

Comment:
Any reason for this change? Was useful to control output graph for readme (and similarly useful to keep the same output)